pkg/config/env.go

```diff
@@ -42,6 +42,9 @@ var Config = struct {
 	// DBConnectionDebug controls whether to show the database connection debugging logs
 	// warning: it may log the credentials to the stdout
 	DBConnectionDebug bool `env:"FLAGR_DB_DBCONNECTION_DEBUG" envDefault:"true"`
+	// DBConnectionRetryAttempts controls how we are going to retry on db connection when start the flagr server
```

Comment:
a note to myself, we probably need to document in the env.go, what are the minimal env variables do we need for a new setup.